server/api/users/users.test.ts

```diff
@@ -207,4 +209,38 @@ describe('user router', () => {
       });
     });
   });
+
+  describe('Create new flag', () => {
+    it('should create a new flag and verify the data', async () => {
+      const newFlag = {
+        name: 'test flag',
+        description: 'this is a test flag',
+        isActive: true,
+      };
+      const response = await axios.post('/api/flags', newFlag);
```

Comment:
does this route exist? i don't think we should have an endpoint that allows arbitrary flag creation - it should be managed solely on the server side